### ISSUE TITLE
Interrupt a file upload

### ISSUE THREAD

**Issue by qgau** [InseeFrLab/onyxia#677]
Hello!

Would it be possible to allow the user to interupt the ongoing upload of a file?

Thank you!

**Comment by fcomte** [comment]
I think it's possible,  @garronej ? 
There is a lot of work on the file browser ( sharing files , listing view etc )
We will look at this when we will review this component in the next few months.

If you want to contribute I guess this feature is not very hard to implement

**Comment by qgau** [comment]
Yes we plan to help indeed, just creating issues in batch to not forget some our user feature requests ;-)

**Comment by fcomte** [comment]
yes this is perfect .
I just wanted to say that File Browser need a lot of work so at our side we will not fix little part soon but refine it globally when first step on data explorer will be released
